Makefile

```diff
@@ -1,6 +1,5 @@
 IMAGE?=copacetic/copacetic-docker-desktop-extension
-TAG?=latest
-COPA_VERSION?=latest
+TAG?=0.1.0
```

Comment:
why did we get rid of COPA_VERSION?

Reply:
not sure how this happened, something might have happened during a merge in the past